libs/blocks/icon-block/icon-block.js

```diff
@@ -15,13 +15,15 @@
 */
 
 import { decorateBlockText, getBlockSize } from '../../utils/decorate.js';
+import { createTag } from '../../utils/utils.js';
 
-const variants = ['fullwidth', 'vertical', 'bio'];
+const variants = ['fullwidth', 'vertical', 'bio', 'icon-inline'];
 const iconBlocks = {
   small: {
     [variants[0]]: ['m', 'm'],
     [variants[1]]: ['s', 'm'],
     [variants[2]]: ['s', 's'],
+    [variants[3]]: ['s', 's'],
```

Comment:
Is this really needed? If yes, an additional entry would be missing for both `medium` and `large`.

Reply:
yes, it is really needed, since we can not use existing variants[2] for inline variation.. Added additional entry for medium and large.